### DISCUSSION TITLE
There is some way to overwrite the MaterialDesignPaper brush

### DISCUSSION THREAD

**Discussion by OrlandoNatsu** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3082] [Q&A] [answered]
I was able to verify that the MaterialDesignPaper brush changes dynamically when you change the mode (light,dark), There is some way to override or change the color of the brush only for the light theme, and keep its color change property?

**Comment by Keboo** [comment, marked as answer]
Yes, this is something that is getting a major re-work done for the 5.0.0 release.

For now the easiest way to do it is to get a hold of the `ITheme` instance, modify the brush, and save the change. The basic code looks like this:

```C#
var paletteHelper = new PaletteHelper();
ITheme theme = paletteHelper.GetTheme();
theme.Paper = Colors.Fuchsia;
paletteHelper.SetTheme(theme);
```

If you let the theme change from user to user you can use the `IThemeManager` to be notified when the theme changes and update the brush. Something like this:
```C#
var paletteHelper = new PaletteHelper();
IThemeManager manager = paletteHelper.GetThemeManager();
manager.ThemeChanged += OnThemeChanged;

UpdateBrushes();

void OnThemeChanged(object? sender, ThemeChangedEventArgs e)
{
    manager.ThemeChanged -= OnThemeChanged;
    UpdateBrushes();
    manager.ThemeChanged += OnThemeChanged;
}

void UpdateBrushes()
{
    ITheme theme = paletteHelper.GetTheme();

    if (theme.GetBaseTheme() == BaseTheme.Light)
    {
        theme.Paper = Colors.Fuchsia;

    }
    paletteHelper.SetTheme(theme);
}
```

**Reply by OrlandoNatsu** [reply]
Thank you very much, it is exactly what I asked, I will try it, but I find it a bit far-fetched. Hopefully in 5.0.0 it will be more natural like the rest of the toolkit. Thank you for your response and for your time.

**Reply by Keboo** [reply]
No problem, certainly trying to make doing stuff like this much easier in 5.0.0
